lib/ddtrace/contrib/sequel/utils.rb

```diff
@@ -0,0 +1,28 @@
+module Datadog
+  module Contrib
+    module Sequel
+      # General purpose functions for Sequel
+      module Utils
+        module_function
+
+        def adapter_name(database)
+          Datadog::Utils::Database.normalize_vendor(database.adapter_scheme.to_s)
+        end
+
+        def parse_opts(sql, opts, db_opts)
+          if ::Sequel::VERSION > '4.36.0' && !sql.is_a?(String)
```

Comment:
I may miss this part, but if the version is `4.36.1` is it ok to access `sql.prepared_sql`? Because from the comment it seems no.